modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -14,6 +14,475 @@
+
+[discrete]
+=== Experimental: Support of AD FS
+
+Experimental support for AD FS has been added. AD FS `/adfs/.well-known/openid-configuration` has an optional `access_token_issuer` which, in violation of the OpenID Connect spec, takes precedence over `issuer`.
+
+[discrete]
+=== Enhanced Extension Capabilities
+
+To enable custom Infinite Scale extensions, custom routes have been added to the Infinite Scale proxy service. More details can be found in the https://owncloud.dev/services/proxy/#configuring-routes[Developer Documentation].
+
+[discrete]
+=== Use Environment Variables in yaml Config Files
+
+The ability to use environment variables in yaml config files has been added to make configuring Infinite Scale services easier. The value in the yaml file will be replaced by the actual value of the environment variable at runtime. This makes it possible to use the same config file for different environments without the need to change the config file itself, useful like when using docker compose with `.env` files to run Infinite Scale services.
+
+[discrete]
+=== Configurable Eventbus
+
+The event bus used in many services is now configurable with a set of environment variables starting with `OCIS_EVENTS_xxx`. This is important for scaling when deploying your instance with an orchestration tool like Docker or Kubernetes. External stores used in caching can be reused for the event bus easying the setup. See the _Environment Variables with Special Scope_ documentation for a list of services affected. Each service listed has a detailed description.
+
+[discrete]
+=== New Services
+
+The following services have been added:
+
+* `auth-service`: +
+The Infinite Scale auth-service is used to authenticate service accounts. Compared to normal accounts, service accounts are Infinite Scale internal and not available to ordinary users like via LDAP. https://github.com/owncloud/ocis/pull/6427[#6427]
+
+* `clientlog`: +
+The Infinite Scale clientlog service is responsible for composing machine-readable notifications for clients. Clients are apps and web interfaces. https://github.com/owncloud/ocis/pull/7217[#7217]
+
+* `ocm`: +
+The Infinite Scale OCM service provides federated sharing functionality based on the ScienceMesh and OCM HTTP APIs. https://github.com/owncloud/ocis/pull/7998[#7998]
+
+* `sse`: +
+The Infinite Scale sse service is responsible for sending sse (server-sent events) to a user. The referenced pull request https://github.com/owncloud/ocis/pull/6992[#6992] is the initial PR introducing SSE. More PRs have been added to improve and extend the SSE service. For details see the Infinite Scale changelog.
+
+=== Known Issues
+
+This section will be updated if issues are discovered.
+
+=== Deprecation Announcements
+
+In future releases the following may no longer be supported or get removed:
+
+* Resharing feature
+* ocs API +
+The ocs service may get fully removed.
```

Comment:
```suggestion
* The resharing feature +
Existing shares will continue to work, but no new reshares can be created.
* The ocs sharing API +
It will be fully replaced by the new sharing-ng (graph) API
* The store service will get fully removed +
Its tasks will be taken over by other services.
```